source/features/tags-dropdown.tsx

```diff
@@ -37,12 +37,16 @@ function onFragmentLoaded(): void {
 	// Change the tab to "Tags"
 	select('.rgh-tags-dropdown .select-menu-tab:last-child button')!.click();
 
-	// Change links, which point to the content of each tag, to open the tag page instead
-	for (const anchorElement of select.all<HTMLAnchorElement>('.rgh-tags-dropdown [href*="/tree/"]')) {
-		const pathnameParts = anchorElement.pathname.split('/');
-		pathnameParts[3] = 'releases/tag'; // Replace `tree`
-		anchorElement.pathname = pathnameParts.join('/');
-	}
+	// https://github.com/github/remote-input-element#events
+	// Wait until the network request is finished and HTML body is updated
+	select('.rgh-tags-dropdown remote-input')!.addEventListener('remote-input-success', () => {
```

Comment:
Can you use `one-event.ts` instead?

Reply:
Good suggestion but `remote-input` also fetches new tags as we type in the input box so this callback should be invoked every time tags are fetched. Making to run it only one time will fail to modify links on the newly fetched data.

Reply:
Hfc